### PR TITLE
feat: add admin module and dashboard component

### PR DESCRIPTION
**Pull request by zaidi-01**
This pull request includes the following changes:

- Remove the login menu

- Add a home component to handle root routing.

- Add an admin module and an admin dashboard component.

These changes improve the overall structure and functionality of the application.

### CODE REVIEW

**Review by NilinReza — APPROVED** [review]
Way to go tiger

**Review by NilinReza — DISMISSED** [review]

File: client/src/app/nav-menu/nav-menu.component.html [hunk 1]
```diff
@@ -21,10 +21,14 @@
       >
         <ul class="navbar-nav flex-grow">
           <li class="nav-item" [routerLinkActive]="['link-active']">
-            <a class="nav-link text-dark" [routerLink]="['/fridge']">Fridge</a>
+            <a
+              class="nav-link text-dark"
+              [routerLink]="['/{{ appPaths.fridge }}']"
```
Comment (JJTomines) [inline client/src/app/nav-menu/nav-menu.component.html:26]: Small bug

Variables for routerLinks are added by doing `[routerLink]="['/' ,appPaths.fridge]"` , received an error while trying to access fridge on nav-menu

File: client/src/api-authorization/api-authorization.module.ts [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
-        { path: ApplicationPaths.LoginFailed, component: LoginComponent },
-        { path: ApplicationPaths.LoginCallback, component: LoginComponent },
-        { path: ApplicationPaths.LogOut, component: LogoutComponent },
-        { path: ApplicationPaths.LoggedOut, component: LogoutComponent },
-        { path: ApplicationPaths.LogOutCallback, component: LogoutComponent }
-      ]
-    )
+    RouterModule.forChild([
+      { path: ApplicationPaths.Register, component: LoginComponent },
+      { path: ApplicationPaths.Profile, component: LoginComponent },
```
Comment (StevieShibly8) [inline client/src/api-authorization/api-authorization.module.ts:15]: Do we need register and profile routes?

**Review by StevieShibly8 — APPROVED** [review]

File: client/src/app/nav-menu/nav-menu.component.html (hunk 1 above)
Comment (zaidi-01) [reply client/src/app/nav-menu/nav-menu.component.html]: Thanks for pointing it out, fixed it.

File: client/src/api-authorization/api-authorization.module.ts (hunk 2 above)
Comment (zaidi-01) [reply client/src/api-authorization/api-authorization.module.ts:15]: Shouldn't need these anymore. I'll target everything authorization API related with login.

**Review by JJTomines — APPROVED** [review]
All good now! :)

**Review by StevieShibly8 — APPROVED** [review]

**Pull request merged**